templates/emails/collective.expense.missing.receipt.hbs

```diff
@@ -1,36 +1,30 @@
-Subject: Action Required: Please submit receipt for card expense
+Subject: Action Required: Submit receipt for card expense
 
 
 {{> header}}
 
 <center>
   <h1>
-    Please submit a receipt for card expense from {{fromCollective.name}}
+    Please submit the receipt for {{fromCollective.name}} card expense.
   </h1>
 </center>
-
+<p>
 {{#if user}}
 A card attached to {{fromCollective.name}} was charged {{currency expense.amount currency=expense.currency
   precision=2}} and we need the responsible admin {{user.firstName}} {{user.lastName}} to input additional
 information.
```

Comment:
@alanna : This has a conflict since the same text was change in the master branch. I've corrected the conflict, but could you take a look specifically at the text which I've highlighted above. This part was added newly and thus you may want to correct the wording there too. 😄

Reply:
Looks good! Let's go for it.